dev/release-aux/openssl-announce-pre-release.tmpl

```diff
@@ -0,0 +1,49 @@
+
+   OpenSSL version $release_text released
+   ======================================
+
+   OpenSSL - The Open Source toolkit for SSL/TLS
+   https://www.openssl.org/
+
+   OpenSSL $series is currently in $label.
+
+   OpenSSL $release_text has now been made available.  For details of
+   changes and known issues see the release notes at:
+
+        https://www.openssl.org/news/openssl-$series-notes.html
```

Comment:
Just to note that this page (perhaps not unsurprisingly) does not yet exist for 3.0. Do we need to update something to make sure that it is?

Reply:
Yes we do, but that's in openssl/web.